src/JustSaying/Messaging/Compression/GzipMessageBodyCompression.cs

```diff
@@ -0,0 +1,48 @@
+using System.IO.Compression;
+using System.Text;
+
+namespace JustSaying.Messaging.Compression;
+
+/// <summary>
+/// Implements GZIP compression and Base64 encoding for message bodies.
+/// </summary>
+public sealed class GzipMessageBodyCompression : IMessageBodyCompression
```

Comment:
Very minor, but is it worth noting in the classname that this class also does b64 encoding?

Reply:
Personally I think this is ok to be implicit